modules/@apostrophecms/schema/ui/apos/mixins/AposInputChoicesMixin.js

```diff
@@ -29,5 +29,24 @@ export default {
     } else {
       this.choices = this.field.choices;
     }
+
+    if (this.field.type === 'select') {
+      this.prependEmptyChoice();
+    }
+  },
+
+  methods: {
+    prependEmptyChoice() {
```

Comment:
fixed: add an empty choice, even for required fields, when field does not have a `def` option.